core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/sql/SQLSourceOpDesc.scala

```diff
@@ -133,8 +133,7 @@ abstract class SQLSourceOpDesc extends SourceOperatorDescriptor {
         val columnName = columns.getString("COLUMN_NAME")
         val datatype = columns.getInt("DATA_TYPE")
         datatype match {
-          case Types.BIT | // -7 Types.BIT
```

Comment:
not sure if this would cause an issue in some applications because in postgres and sql server bit is truly treated as boolean but mysql might have different usage https://dev.mysql.com/doc/refman/8.0/en/bit-type.html

Reply:
true, I've seen MySQL supports `BIT(M)` type, which is an m-bit value. But I think such type is rarely being used, we can find a workaround for that `BIT(M)` type later.